Scripts/build

```diff
@@ -70,8 +77,33 @@ customScalars = [
 
 content = GraphQLSwiftGen.new(
 	schema,
-	nest_under: 'Storefront',
+	nest_under: generated_nesting_class,
 	import_graphql_support: false,
 	custom_scalars: customScalars
 )
-content.save("../Buy/Generated/")
+content.save(generated_code_path)
+
+
+################################################
+# Add newly generated sources to xcode project
+################################################
+
+project_path = File.join(parent_path, 'Buy.xcodeproj')
+project = Xcodeproj::Project.open(project_path)
+
+generated_files = Dir[File.join(generated_code_path, generated_nesting_class, '*')]
+
+buy_target = project.targets.find{ |t| t.name == 'Buy'}
```

Comment:
This gets the tv and watch targets too?

Reply:
@DrTeePot Actually you are right. The generated files are required for all Buy targets. Will update that